torch/nn/_functions/rnn.py

```diff
@@ -391,6 +395,43 @@ def reform(x):
     return torch.onnx.symbolic_override(symbolic)
 
 
+def GRU_symbolic_builder(input_size, hidden_size, num_layers, batch_first, dropout, bidirectional, **kwargs):
```

Comment:
Can we put those somewhere in ONNX dir? This is the code that implements RNNs and it's better to not mix them up